github_webhook/webhook.py

```diff
@@ -69,7 +69,9 @@ def _postreceive(self):
             '%s (%s)', _format_event(event_type, data), _get_header('X-Github-Delivery'))
 
         for hook in self._hooks.get(event_type, []):
-            hook(data)
+            resp = hook(data)
```

Comment:
If the use case is sending an error, would this be better represented with an exception?

Reply:
In this case we're returning the http code that the wrapped function returns.  This could be any http code such as a 301 which is not necessarily an error.  I think the current approach is the best choice here, but I'm happy to be corrected.